wp-content/themes/engage/templates/page-about.twig

```diff
@@ -1,22 +1,29 @@
 {% extends "base.twig" %}
 
 {% block content %}
-	<article class="container container--md post-type--{{post.post_type}}" id="post--{{post.ID}}">
-		<header class="article__header">
-			<h1 class="article__title">{{post.title}}</h1>
-		</header>
-		<div class="article__content">
-			{{post.content}}
-		</div>
-		<div class="newsletter__wrapper">
-		  <div class="newsletter-page">
-		      {{ newsletter }}
-		  </div>
-		</div>
-		<div class="home-funders home-section">
-			<div class="container container--xl wrapper--pad">
-				{% include 'partial/funders.twig' with { funders: about.funders } %}
-			</div>
-		</div>
-	</article>
-{% endblock %}
+    <article class="container container--md post-type--{{post.post_type}}" id="post--{{post.ID}}">
+        <header class="article__header">
+            <h1 class="article__title">{{post.title}}</h1>
+        </header>
+    </article>
+    <article class="container container--lg post-type--{{post.post_type}}" id="post--{{post.ID}}">
```

Comment:
The `<article>` HTML tag should only be on here once, as it indicates that the content all goes together. Right now the document looks like it has two articles, one with a title, and one with content. We want those all together under one `<article>` tag.